include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //uint32_t           maxTransformFeedbackBufferDataStride;
+    //VkBool32           transformFeedbackQueries;
+    //VkBool32           transformFeedbackStreamsLinesTriangles;
+    //VkBool32           transformFeedbackRasterizationStreamSelect;
+    //VkBool32           transformFeedbackDraw;
+
+    /* VertexAttributeDivisorPropertiesEXT *//* provided by VK_EXT_vertex_attribute_divisor */
+    //uint32_t           maxVertexAttribDivisor;
+
+    /* AccelerationStructurePropertiesKHR *//* provided by VK_KHR_acceleration_structure */
+    uint64_t           maxGeometryCount;
+    uint64_t           maxInstanceCount;
+    uint64_t           maxPrimitiveCount;
+    uint32_t           maxPerStageDescriptorAccelerationStructures;
+    uint32_t           maxPerStageDescriptorUpdateAfterBindAccelerationStructures;
+    uint32_t           maxDescriptorSetAccelerationStructures;
+    uint32_t           maxDescriptorSetUpdateAfterBindAccelerationStructures;
+    uint32_t           minAccelerationStructureScratchOffsetAlignment;
+ 
+    /* DepthStencilResolvePropertiesKHR ---> MOVED TO Vulkan 1.2 Core  */
+    /* DriverPropertiesKHR ---> MOVED TO Vulkan 1.2 Core  */
+    /* VK_KHR_fragment_shader_barycentric --> Coverage 0% --> no structs defined anywhere in vulkan headers */
+
+    /* FragmentShadingRatePropertiesKHR *//* provided by VK_KHR_fragment_shading_rate */
+    //VkExtent2D               minFragmentShadingRateAttachmentTexelSize;
+    //VkExtent2D               maxFragmentShadingRateAttachmentTexelSize;
+    //uint32_t                 maxFragmentShadingRateAttachmentTexelSizeAspectRatio;
+    //VkBool32                 primitiveFragmentShadingRateWithMultipleViewports;
+    //VkBool32                 layeredShadingRateAttachments;
+    //VkBool32                 fragmentShadingRateNonTrivialCombinerOps;
+    //VkExtent2D               maxFragmentSize;
+    //uint32_t                 maxFragmentSizeAspectRatio;
+    //uint32_t                 maxFragmentShadingRateCoverageSamples;
+    //VkSampleCountFlagBits    maxFragmentShadingRateRasterizationSamples;
+    //VkBool32                 fragmentShadingRateWithShaderDepthStencilWrites;
+    //VkBool32                 fragmentShadingRateWithSampleMask;
+    //VkBool32                 fragmentShadingRateWithShaderSampleMask;
+    //VkBool32                 fragmentShadingRateWithConservativeRasterization;
+    //VkBool32                 fragmentShadingRateWithFragmentShaderInterlock;
+    //VkBool32                 fragmentShadingRateWithCustomSampleLocations;
+    //VkBool32                 fragmentShadingRateStrictMultiplyCombiner;
+    
+    /* Maintenance2PropertiesKHR *//* provided by VK_KHR_maintenance2 *//* MOVED TO Vulkan 1.1 Core  */
+    /* Maintenance3PropertiesKHR *//* provided by VK_KHR_maintenance3 *//* MOVED TO Vulkan 1.1 Core  */
+    /* Maintenance4PropertiesKHR *//* provided by VK_KHR_maintenance4 *//* MOVED TO Vulkan 1.3 Core  */
```

Comment:
our minimum is 1.1 so we need to care about contents of `Maintenance4PropertiesKHR `

Reply:
It is moved to Vulkan 1.1 feature/limit structs, we don't need specific extensions to query for vulkan 1.1, It's all there above